.github/CODEOWNERS

```
@@ -30,7 +30,9 @@
 /sdk/tests/iot/                @CIPop @ericwol-msft @ewertons @jspaith @preethi826 @RLeclair @vaavva
```

```suggestion
/sdk/docs/iot/                 @CIPop @ericwol-msft @ewertons @jspaith @RLeclair @vaavva
/sdk/inc/azure/iot/            @CIPop @ericwol-msft @ewertons @jspaith @RLeclair @vaavva
/sdk/samples/iot/              @CIPop @ericwol-msft @ewertons @jspaith @RLeclair @vaavva
/sdk/src/azure/iot/            @CIPop @ericwol-msft @ewertons @jspaith @RLeclair @vaavva
/sdk/tests/iot/                @CIPop @ericwol-msft @ewertons @jspaith @RLeclair @vaavva
```

While you're already in here editing the codeowners file